src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -98,13 +98,13 @@ public CommandResult execute(Model model) throws CommandException {
         if (!personToEdit.isSamePerson(editedPerson) && model.hasPerson(editedPerson)) {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         }
-        boolean isEditingClass = !editPersonDescriptor.getAClass().get().classDateTime.equals("");
+        boolean isEditingClass = !editPersonDescriptor.getAClass().get()
+                .classDateTime.equals("");
```

Comment:
This line is a bit long and seems to be violating the Laws of Demeter. Could be better if Person contains a hasEmptyClass() which calls Class to check whether it equals to ("").